Description: Fix FTBFS due missing unistd.h include
 Probably caused by a more recent C/C++ compiler
Bug-Debian: https://bugs.debian.org/1011681
Author: Axel Beckert <abe@debian.org>

--- a/src/generic/apt/dpkg.cc
+++ b/src/generic/apt/dpkg.cc
@@ -35,6 +35,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <fcntl.h>
+#include <unistd.h>
 
 using namespace std;
 
